frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/reducer.ts

```diff
@@ -56,6 +65,9 @@ export const initialState: InternalClusterState = {
   nodes: [],
   enableMinimal: false,
   enableFlexibleScaling: false,
+  enableStretchCluster: false,
+  selectedArbiterZone: '',
+  isArbiterValid: false,
```

Comment:
helps to identify the usage of states. Lets follow the same for arbiter.
```suggestion
// arbiter state
  enableStretchCluster: false,
  selectedArbiterZone: '',
  isArbiterValid: false,
```